resources/core/values.yaml

```
@@ -21,7 +21,7 @@ global:
     # - Change the value below to the branch from your fork if you want to point to documentation sources from your fork
     # - Change the value below to the release branch during the release
     # Example: clusterAssetGroupsVersion: "release-0.9"
-    clusterAssetGroupsVersion: master
+    clusterAssetGroupsVersion: release-1.16.0-rc1
```

here it's enough to have the "main" release version
```suggestion
    clusterAssetGroupsVersion: release-1.16
```